src/Parser/Warlock/Affliction/Modules/Items/Tier21_4set.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+import ItemDamageDone from 'Main/ItemDamageDone';
+import { formatNumber, formatPercentage } from 'common/format';
+import getDamageBonus from 'Parser/Warlock/Affliction/Modules/WarlockCore/getDamageBonus';
+
+const AFFECTED_ABILITIES = new Set([
```

Comment:
My understanding is that for small collections, just keeping this as an array and using `includes` is actually more performant than using a Set.